rollup/core/classes/RollupAsyncProcessor.cls

```diff
@@ -509,6 +564,18 @@ global virtual without sharing class RollupAsyncProcessor extends Rollup impleme
       }
     }
     localLookupItems.addAll(roll.getExistingLookupItems(lookupItemKeys, roll, this.lookupObjectToUniqueFieldNames.get(roll.lookupObj)));
+    // TODO - come back to this and last test in InvocableDrivenTests
```

Comment:
this is also part of the unfinished work for #149 - happy to remove it, but I'll wait for other feedback first so as to not run through too many package versions.